### PR TITLE
Fix the Domains page view on the before-login view

### PR DESCRIPTION
**Pull request by samaradel**
### Description

Fix the Domains page view on the before-login view
### Changes

[Screencast from 16-10-24 14:45:32.webm](https://github.com/user-attachments/assets/88b5fe45-6fbe-4292-8ea1-92ae5c5ec05c)

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3520

### Tested Scenarios

- Try to reach the domains page before logging in.
- After login go to the domains page and logout from there.

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by 0oM4R — APPROVED** [review]

File: packages/playground/src/components/node_selector/TfDomainName.vue [hunk 1]
```diff
@@ -147,7 +147,6 @@ export default {
       size: window.env.PAGE_SIZE,
       page: Math.max(1, pagination.value.page),
       farmId: enableCustomDomain.value ? props.farm?.farmId : undefined,
-      availableFor: gridStore.client.twinId,
```
Comment (0oM4R) [inline packages/playground/src/components/node_selector/TfDomainName.vue:150]: why did you remove this ?

**Review by 0oM4R — CHANGES_REQUESTED** [review]

File: packages/playground/src/components/node_selector/TfDomainName.vue (hunk 1 above)
Comment (samaradel) [reply packages/playground/src/components/node_selector/TfDomainName.vue:150]: It's seeking for the twinId when you logout from the dashboard when you are already on the domains page and the page blocks on that error so you have to refresh the page to apply the logout method.

Comment (0oM4R) [reply packages/playground/src/components/node_selector/TfDomainName.vue:150]: but i think this will break the filter logic!, available for is needed while listing the nodes in the domains

Comment (samaradel) [reply packages/playground/src/components/node_selector/TfDomainName.vue:150]: maybe it needs to be optional to avoid breaking on logout, let me check

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]
breaks when you do the following:
- log in
- go to domains page
- press log out button
- look at console error below

image

**Review by amiraabouhadid — APPROVED** [review]
my bad was testing wrong branch lgtm

**Review by maayarosama — APPROVED** [review]

**Review by 0oM4R — APPROVED** [review]
